src/raylib/core/Window.cpp

```diff
@@ -0,0 +1,36 @@
+/*
+** EPITECH PROJECT, 2022
+** Bomberman-Global-Offensive
+** File description:
+** Window
+*/
+
+#include "Window.hpp"
+
+extern "C"
+{
+#include "raylib.h"
+}
+
+namespace raylib
+{
+    namespace core
+    {
+        void Window::open(int width, int height, const char *title) { InitWindow(width, height, title); }
+
+        void Window::close() { CloseWindow(); }
+
+        void Window::clear() { ClearBackground(BLACK); }
```

Comment:
A Color parameter could be needed sometimes, make it with a default value of Black